configs/project-refs.conf

```diff
@@ -59,7 +59,10 @@ vars.uris: {
   play-ws-uri:                  "https://github.com/playframework/play-ws.git#8011fb8"  # was master
   pprint-uri:                   "https://github.com/lihaoyi/pprint.git"
   sbinary-uri:                  "https://github.com/scalacommunitybuild/sbinary.git#community-build-2.12"
-  sbt-testng-uri:               "https://github.com/scalacommunitybuild/sbt-testng.git#community-build-2.12"
+  sbt-uri:                      "https://github.com/sbt/sbt.git#1.0.0"
+  sbt-io-uri:                   "https://github.com/sbt/io.git#v1.0.0-M11"
+  sbt-librarymanagement-uri:    "https://github.com/sbt/librarymanagement.git#v1.0.0-X15"
+  sbt-util-uri:                 "https://github.com/sbt/util.git#v1.0.0-M24"
```

Comment:
this is blocked on sbt 1.0 support in dbuild, so considering it on hold for now.

but, I will separately try to add zinc.